### PR TITLE
[patch] version is not added on released steps

### PR DESCRIPTION
**Pull request by WindzCUHK**
similar PR: https://github.com/yahoojapan/athenz-client-sidecar/pull/29

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -53,7 +53,12 @@ jobs:
       - run:
           name: docker image build
           command: |
-            TAG=`cat ./.tag`
+            if [ ! -z "$CIRCLE_TAG" ]; then
```
Comment (kevindiu) [inline .circleci/config.yml]: should the logic is something like 

```
if ./.tag file found 
    use the tag file version
else 
    ?? not sure the handling 
```

Comment (WindzCUHK) [reply .circleci/config.yml]: CIRCLE_TAG has priority.
On `./.tag` not found,`TAG` is empty.
the below if else will do the checking

Comment (kevindiu) [reply .circleci/config.yml]: ok

**Comment by kevindiu** [comment]
LGTM

**Pull request merged**